.travis.yml

```diff
@@ -2,5 +2,7 @@ language: node_js
 node_js:
   - 0.8
   - 0.10
+  - 0.12
+  - iojs
 before_install:
   - npm install -g npm@~1.4.6
```

Comment:
On a tangential note, is there a specific reason to stay with [npm@1.4.6](https://github.com/npm/npm/releases/tag/v1.4.6)?
It's almost a year old and [npm's](https://www.npmjs.com/package/npm) current requirement of node >= 0.8 seems like a non-issue no?